src/GameWorldSimulator/NeoServer.Game.Common/Contracts/DataStores/IVocationStore.cs

```diff
@@ -1,7 +1,12 @@
 ﻿using NeoServer.Game.Common.Contracts.Creatures;
+using System.Linq;
 
 namespace NeoServer.Game.Common.Contracts.DataStores;
 
 public interface IVocationStore : IDataStore<byte, IVocation>, IDataStore
 {
+    public virtual IVocation GetByName(string name)
+    {
+        return All.FirstOrDefault(c => c.Name.ToLower().Equals(name.ToLower()));
```

Comment:
If this method will be called prequently let's use a foreach loop instead of Linq.
Also change the equals to use InvariantCase:
c.Name.Equals(name, InvariantCase)

Reply:
removed on last commit: https://github.com/OpenCoreMMO/OpenCoreMMO/pull/665/commits/82314937fa9eeedb50263b395791ec30a500404b